apps/portals/adknowledgeportal/src/config/doiRedirectConfig.ts

```diff
@@ -0,0 +1,17 @@
+import { DoiRedirectConfig } from '@sage-bionetworks/synapse-portal-framework/shared-config/DoiRedirectRoute'
+
+type ADKPResourceType = 'STUDY'
+
+export const doiRedirectConfig: DoiRedirectConfig<ADKPResourceType> = (
+  resourceType,
+  resourceId,
+) => {
+  switch (resourceType) {
+    case 'STUDY':
+      return `/Explore/Studies/DetailsPage/StudyDetails?Study=${resourceId}`
+    default:
+      throw new Error(
+        `Resource type ${resourceType} not supported in this portal.`,
+      )
+  }
+}
```

Comment:
Each portal must define a mapping from `(resourceType, resourceId)  -> path`. I started one for the ADKP here.

Reply:
I need to reach out to the ADKP stakeholders to make sure `Study` is the correct column to use for a permanent identifier, so the implementation is subject to change.